conf/battle/feature.conf

```
@@ -33,6 +33,11 @@ feature.warp_suggestions: off
 // Requires: 2013-07-24aRagexe or later
 feature.banking: on
 
+// Should Banking strictly checks the UI state on zeny deposit and withdrawal? (Note 1)
+// Note: Disabled by default because existing official clients do not report the banking UI state to the server.
+// But this config could be useful in case of clients that actually report the UI state or custom clients.
+feature.banking_state_enforce: off
```

```suggestion
feature.banking_state_enforce: no
```
Just sounds better to me.

---

It is best to use the usual conventions used in other confs, which are usually yes and no.